src/Core/Utils/Timer.hpp

```diff
@@ -1,31 +1,31 @@
-#ifndef RADIUMENGINE_TIMER_HPP_
-#define RADIUMENGINE_TIMER_HPP_
-
-#include <Core/RaCore.hpp>
-#include <chrono>
-
-namespace Ra {
-namespace Core {
-// Convenience functions for timing.
-// Wrapping the highest resolution time measuring.
-namespace Timer {
-using MicroSeconds = long;
-using Seconds = Scalar;
-using Clock = std::chrono::high_resolution_clock;
-using TimePoint = std::chrono::time_point<Clock>;
-
-inline MicroSeconds getIntervalMicro( const TimePoint& start, const TimePoint& end ) {
-    return ( std::chrono::duration_cast<std::chrono::microseconds>( end - start ) ).count();
-}
-
-inline Seconds getIntervalSeconds( const TimePoint& start, const TimePoint& end ) {
-    return ( std::chrono::duration_cast<
-                 std::chrono::duration<Scalar, std::chrono::seconds::period>>( end - start ) )
-        .count();
-}
-} // namespace Timer
-
-} // namespace Core
-} // namespace Ra
-
-#endif // RADIUMENGINE_TIMER_HPP_
+#ifndef RADIUMENGINE_TIMER_HPP_
+#define RADIUMENGINE_TIMER_HPP_
+
+#include <Core/RaCore.hpp>
+#include <chrono>
+
+namespace Ra {
+namespace Core {
+// Convenience functions for timing.
+// Wrapping the highest resolution time measuring.
+namespace Utils {
+using MicroSeconds = long;
```

Comment:
All the above should be in a Timer class with static member function.
This will allow to have namespace coherent with directory structure and access coherent with semantic 
So Ra::Core::Utils is the namespace
::Timer is the static class container.
@louen @nmellado feel free to argue/comment if I'm wrong wrt radium coding style

Reply:
:+1: